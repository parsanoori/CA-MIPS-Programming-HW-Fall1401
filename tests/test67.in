18
418085178
376314739
367740461
28984388
111074129
430044900
284093799
48749785
502487566
969493406
1023279573
668544945
888025680
926832063
224087967
171906047
916424938
88127890