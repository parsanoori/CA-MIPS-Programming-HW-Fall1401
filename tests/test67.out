28984388
48749785
88127890
111074129
171906047
224087967
284093799
367740461
376314739
418085178
430044900
502487566
668544945
888025680
916424938
926832063
969493406
1023279573